pkg/analyzer/analyzer_test.go

```diff
@@ -1,11 +1,12 @@
 package analyzer_test
 
 import (
-	"github.com/Crocmagnon/fatcontext/pkg/analyzer"
-	"golang.org/x/tools/go/analysis/analysistest"
 	"os"
 	"path/filepath"
 	"testing"
+
+	"github.com/Crocmagnon/fatcontext/pkg/analyzer"
+	"golang.org/x/tools/go/analysis/analysistest"
```

Comment:
I tend to use this format

```suggestion
 	"os"
 	"path/filepath"
 	"testing"

	"golang.org/x/tools/go/analysis/analysistest"

	"github.com/Crocmagnon/fatcontext/pkg/analyzer"
```

```shell
gci write -s standard -s default -s local module
```

Also you can configure golangci-lint

https://github.com/ccoVeille/golangci-lint-config-examples/blob/main/03-safe%2F.golangci.yml#L52-L60

The simply use

```shell
golangci-lint run --fix --enable-only gci
```